crates/uv-resolver/src/prerelease.rs

```diff
@@ -84,7 +84,10 @@ impl PrereleaseStrategy {
 
                     if specifier
                         .iter()
-                        .filter(|spec| (spec.operator() != &Operator::NotEqual))
+                        .filter(|spec| {
+                            (spec.operator() != &Operator::NotEqual)
+                                && (spec.operator() != &Operator::NotEqualStar)
```

Comment:
You might be interested in something like: `!matches!(spec.operator(), Operator::NotEqual | Operator::NotEqualStar)`

Reply:
thanks